app/src/main/java/com/xmartlabs/template/App.kt

```diff
@@ -82,17 +81,20 @@ open class App : Application(), HasActivityInjector {
       .application(this)
       .buildInfo(BuildInfo())
       .okHttpModule(OkHttpModule())
+      .restServiceGsonModule(ServiceGsonModule())
       .restServiceModule(RestServiceModule())
       .build()
 
-  private fun initializeDataBase() = FlowManager.init(FlowConfig.Builder(this).build())
-
   private fun initializeThreeTenABP() = AndroidThreeTen.init(this)
 
   private fun initializeLogging() {
     //TODO: Configure Fabric and add Fabric apiSecret and apiKey properties file in the root folder
     loggerTree.addLogger(CrashlyticsLogger().initialize(buildInfo, this))
     Timber.plant(loggerTree)
+
+    if (buildInfo.isDebug) {
+      Stetho.initializeWithDefaults(this);
```

Comment:
Have you tried Steho? is it good?

Reply:
yep, it's good.

Reply:
but what it's a good feature of it?

Reply:
It provides a simple way of inspect the network queries and the database queries